bigbluebutton-html5/imports/ui/components/presentation/component.jsx

```diff
@@ -274,7 +274,9 @@ class Presentation extends PureComponent {
                 this.currentPresentationToastId = null;
               },
               autoClose: shouldCloseToast,
-              className: 'actionToast currentPresentationToast',
+              className: 'toastClass actionToast',
```

Comment:
currentPresentationToast class is used in the automated tests, we need to keep it

```suggestion
              className: 'toastClass actionToast currentPresentationToast',
```